test/unit_tests/datautil/test_serialization.py

```diff
@@ -115,13 +108,71 @@ def test_load_multiple_concat_windows_dataset(setup_concat_windows_dataset,
     for i in range(2):
         path = os.path.join(tmpdir, str(i))
         os.makedirs(path)
-        save_concat_dataset(path=path,
-                            concat_dataset=concat_windows_dataset,
-                            overwrite=False)
+        concat_windows_dataset._outdated_save(path=path, overwrite=False)
     loaded_concat_windows_datasets = load_concat_dataset(
         path=tmpdir, preload=False)
     assert 2 * len(concat_windows_dataset) == len(loaded_concat_windows_datasets)
     assert (2 * len(concat_windows_dataset.datasets) ==
             len(loaded_concat_windows_datasets.datasets))
     assert (2 * len(concat_windows_dataset.description) ==
             len(loaded_concat_windows_datasets.description))
+
+
+def test_save_concat_raw_dataset(setup_concat_raw_dataset, tmpdir):
+    concat_raw_dataset = setup_concat_raw_dataset
+    n_raw_datasets = len(concat_raw_dataset.datasets)
+    concat_raw_dataset.save(path=tmpdir, overwrite=False)
```

Comment:
Should we add a test for `overwrite=True`? Also, we could also have a test for the case where raw data was saved first, followed by epoched data in the same directory (and vice versa).

Reply:
Yes, we should test it. It feels very error-prone.